go-controller/pkg/ovn/egressgw.go

```diff
@@ -165,7 +181,6 @@ func (oc *Controller) addPodExternalGW(pod *kapi.Pod) error {
 		klog.Warningf("No valid gateway IPs found for requested external gateway pod: %s", pod.Name)
 		return nil
 	}
-
```

Comment:
@tssurya any reason to remove the space, or just left-over change from a previous version of the patch?